integreat_cms/cms/models/pages/page.py

```diff
@@ -310,6 +310,16 @@ def relative_depth(self) -> int:
             return self._relative_depth
         return self.depth
 
+    def can_be_deleted(self) -> tuple[bool, str | None]:
+        """
+        checks if the page can be deleted
+        """
+        if self.children.exists():
+            return False, _("you cannot delete a page which has subpages.")
+        if self.mirroring_pages.exists():
+            return False, _("it was embedded as live content from another page.")
```

Comment:
This can overwrite `error_msg` too.

How to reproduce: choose "Kontakt zu App Team Augsburg" and "Behörden" in Augsburg and bulk delete.